packages/gitgraph-core/src/utils.ts

```diff
@@ -193,26 +194,24 @@ function getAlpha<TNode = SVGElement>(
       break;
   }
 
+  // If commit is distant from its parent, there should be no angle.
```

Comment:
At first, I thought this applied only to reverse arrow scenarios. But it turns out I was wrong. Now it works for every scenario.

Reply:
cc @fabien0102 ;-)